editor/plugins/font_config_plugin.cpp

```diff
@@ -0,0 +1,1001 @@
+	emit_signal(SNAME("object_id_selected"), p_property, p_id);
+}
+
+void EditorPropertyFontMetaOverride::update_property() {
+	Variant updated_val = get_edited_object()->get(get_edited_property());
+
+	Dictionary dict = updated_val;
+
+	edit->set_text(vformat(TTR("Overrides (%d)"), dict.size()));
+
+	bool unfolded = get_edited_object()->editor_is_section_unfolded(get_edited_property());
+	if (edit->is_pressed() != unfolded) {
+		edit->set_pressed(unfolded);
+	}
+
+	if (unfolded) {
+		updating = true;
+
+		if (!vbox) {
+			vbox = memnew(VBoxContainer);
+			add_child(vbox);
+			set_bottom_editor(vbox);
+
+			property_vbox = memnew(VBoxContainer);
+			property_vbox->set_h_size_flags(SIZE_EXPAND_FILL);
+			vbox->add_child(property_vbox);
+
+			paginator = memnew(EditorPaginator);
+			paginator->connect("page_changed", callable_mp(this, &EditorPropertyFontMetaOverride::_page_changed));
+			vbox->add_child(paginator);
+		} else {
+			// Queue children for deletion, deleting immediately might cause errors.
+			for (int i = property_vbox->get_child_count() - 1; i >= 0; i--) {
+				property_vbox->get_child(i)->queue_delete();
+			}
+			button_add = nullptr;
+		}
+
+		int size = dict.size();
+
+		int max_page = MAX(0, size - 1) / page_length;
+		page_index = MIN(page_index, max_page);
+
+		paginator->update(page_index, max_page);
+		paginator->set_visible(max_page > 0);
+
+		int offset = page_index * page_length;
+
+		int amount = MIN(size - offset, page_length);
+
+		dict = dict.duplicate();
+		object->set_dict(dict);
+
+		for (int i = 0; i < amount; i++) {
+			String name = dict.get_key_at_index(i);
+			EditorProperty *prop = memnew(EditorPropertyCheck);
+			prop->set_object_and_property(object.ptr(), "keys/" + name);
+
+			if (script_editor) {
+				prop->set_label(TranslationServer::get_singleton()->get_script_name(name));
+			} else {
+				prop->set_label(TranslationServer::get_singleton()->get_locale_name(name));
+			}
+			prop->set_tooltip(name);
+			prop->set_selectable(false);
+
+			prop->connect("property_changed", callable_mp(this, &EditorPropertyFontMetaOverride::_property_changed));
+			prop->connect("object_id_selected", callable_mp(this, &EditorPropertyFontMetaOverride::_object_id_selected));
+
+			HBoxContainer *hbox = memnew(HBoxContainer);
+			property_vbox->add_child(hbox);
+			hbox->add_child(prop);
+			prop->set_h_size_flags(SIZE_EXPAND_FILL);
+			Button *remove = memnew(Button);
+			remove->set_icon(get_theme_icon(SNAME("Remove"), SNAME("EditorIcons")));
+			hbox->add_child(remove);
+			remove->connect("pressed", callable_mp(this, &EditorPropertyFontMetaOverride::_remove), varray(remove, name));
+
+			prop->update_property();
+		}
+
+		PanelContainer *pc = memnew(PanelContainer);
+		property_vbox->add_child(pc);
+		pc->add_theme_style_override(SNAME("panel"), get_theme_stylebox(SNAME("DictionaryAddItem"), SNAME("EditorStyles")));
+
+		VBoxContainer *add_vbox = memnew(VBoxContainer);
+		pc->add_child(add_vbox);
```

Comment:
```suggestion
```

These suggestions remove the extra background panel from the OpenType Features editor button. 

Changes it from this:
![image](https://user-images.githubusercontent.com/67974470/173170211-57f6015d-befd-45dd-b324-55a9fe2bb3a6.png)

To look more like this:
![image](https://user-images.githubusercontent.com/67974470/173170283-eed575e6-19a1-4e9e-afae-9588f1e71004.png)